datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -288,9 +292,12 @@ private StackV4Request getStackRequest(SdxClusterRequest sdxClusterRequest, Stac
                 throw new BadRequestException("Can't find template for cloudplatform: " + cloudPlatform + ", shape: " + sdxClusterRequest.getClusterShape() +
                         ", runtime version: " + runtimeVersion);
             }
+            updateStackV4RequestWithIDBrokerVolume(stackRequest);
             stackRequest.getCluster().setRangerRazEnabled(sdxClusterRequest.isEnableRangerRaz());
             return stackRequest;
         } else {
+            updateStackV4RequestWithIDBrokerVolume(internalStackV4Request);
```

Comment:
I wonder if anyone relying on custom DL templates would be pissed about this change if they expected the volume to be present (for whatever reason). I know we are already committing such a change by overwriting the RAZ flag just a few lines below. 😄